app/src/main/java/org/imozerov/streetartview/ui/detail/ArtObjectDetailOpener.java

```diff
@@ -0,0 +1,8 @@
+package org.imozerov.streetartview.ui.detail;
+
+/**
+ * Created by sergei on 10.02.16.
+ */
+public interface ArtObjectDetailOpener {
+    void openDetailActivity(String id);
```

Comment:
IMHO 
`openArtObjectDetails(String id) `
will be better name. as activity is part of implementation but not the interface